include/rdma/fi_eq.h

```diff
@@ -76,8 +78,13 @@ struct fid_wait {
 };
 
 struct fi_mutex_cond {
+#ifndef _WIN32
 	pthread_mutex_t		*mutex;
 	pthread_cond_t		*cond;
+#else /* _WIN32 */
+	CRITICAL_SECTION	*mutex;
+	CONDITION_VARIABLE	*cond;
+#endif /* _WIN32 */
```

Comment:
> In include/rdma/fi_eq.h
> <https://github.com/ofiwg/libfabric/pull/2100#discussion_r67316498> :
> 
> >  	pthread_mutex_t		*mutex;
> >  	pthread_cond_t		*cond;
> > +#else /* _WIN32 */
> > +	CRITICAL_SECTION	*mutex;
> > +	CONDITION_VARIABLE	*cond;
> > +#endif /* _WIN32 */
> 
> then we will have to use #ifdefs every time when this structure is used

Correct - the alternative is for the windows implementation to use an actual mutex, rather than pretend that a critical section is one.  This structure is exposed to applications, so it needs to be 'correct' and not lead to developer surprises.  If needed, update this structure and define a new structure.